assets/account/src/lib.rs

```diff
@@ -3,21 +3,21 @@ use scrypto::prelude::*;
 blueprint! {
     struct Account {
         public_key: EcdsaPublicKey,
-        vaults: LazyMap<Address, Vault>,
+        vaults: LazyMap<ResourceDefRef, Vault>,
```

Comment:
This dates back to our original discussion about **resource** and **resource definition**. I'm worried that `ResourceRef` will cause even more confusion.

I agree that the **`ResourceDefRef`** is a bit overwhelming, but if we can come up with _a simpler name for resource definition_, it will start to make sense.